extensions/exposition/source/HTTP/messages.ts

```diff
@@ -4,23 +4,25 @@ import { type Request, type Response } from 'express'
 import { buffer } from '@toa.io/streams'
 import { formats } from './formats'
 import { BadRequest, NotAcceptable, UnsupportedMediaType } from './exceptions'
-import type { ParsedQs } from 'qs'
 import type { Format } from './formats'
+import type { Timing } from './Timing'
+import type { ParsedQs } from 'qs'
```

Comment:
NIT: unrelated to this PR, but we should drop this dependency and use standard `URLSearchParams` for parsing

Reply:
It's not used in actual code.

This type is imported to extend the `express.Request` (which internally uses `qs`). 

It will be eliminated once `express` itself is removed.